org.nifty-site-manager.nsm.json

```diff
@@ -0,0 +1,24 @@
+{
+    "app-id": "org.flatpak.nsm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "nsm",
+    "modules": [
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
```

Comment:
I already gave you the correct snippet here:

```json
{
  "name": "nsm",
  "no-autogen": true,
  "source": ...
}
```

You don't need to run commands manually.

Reply:
I'm not quite sure what you're suggesting here? 

Replace buildsystem with no-autogen and sources? Is that ... for sources what I put or am I meant to know what to put there? Sorry I have not released on FlatHub before..

Reply:
Oh wait, get rid of buildsystem and build-commands?

Reply:
I'm saying remove "buildsystem" and "build-commands", add "no-autogen".